db_test.go

```diff
@@ -437,3 +438,57 @@ func assertContains[K fmt.Stringer](s *SurrealDBTestSuite, input []K, matcher fu
 	s.NotEmptyf(matching, "Input %+v did not contain matching element", fmt.Sprintf("%+v", input))
 	return matching
 }
+
+func TestIsDuplicateUniqueIdx(t *testing.T) {
+	type args struct {
```

Comment:
same with other struct review

Reply:
Yeah, my bad, didn't really take the time to properly write the test with Testify. I'm going to work on it and send another commit. Thanks for your advice!